src/Docker/Client/Utils.hs

```diff
@@ -129,3 +133,16 @@ exclusionCheck f ps = any id (map (\(ExclusionPattern p) -> f ~~ T.unpack p) ps)
 inclusionCheck :: FilePath -> [InclusionPattern] -> Bool
 inclusionCheck f ps = any id (map (\(InclusionPattern p) -> f ~~ T.unpack p) ps)
 
+
+processLog :: Monad m => ConduitT BSS.ByteString BSS.ByteString m ()
+processLog = do
+        -- metadata (is the next string stdout or stderr)
+        _ <- CB.take 4
+        len' <- CB.take 4
```

Comment:
Why do we only take 4 bytes at a time?

Reply:
You mean, why don't we read the first 8 bytes in one go?

Reply:
I'm not sure reading it in one go and partitioning into two parts would be terribly more efficient.